src/main/java/com/bootcamp/demo/business/AccountService.java

```diff
@@ -68,9 +86,14 @@ public boolean accountValidation(Account account){
     public String add(final Account account) throws ServiceException {
         try {
```

Comment:
give Vavr.Try a look. It might improve the readability.
https://www.baeldung.com/vavr-try